utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
```

Comment:
lets rename it to `utils/hash.go`

Reply:
Reasoning: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
done